docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -8,44 +8,30 @@ for examples using the
 [SNS tokenomics training & tools](https://wiki.internetcomputer.org/wiki/How-To:_SNS_tokenomics_configuration)
 as described in [this section](./tokenomics-intro.md).
 After deciding on these configurations, the next step is to
-translate them into 
-[initial parameters](../launching/launch-steps.md/#SNS-launch-step-preparation) <!-- NEW: UPDATE LINK --> that the SNS will be initialized with during the SNS launch.
+translate them into initial parameters that the SNS will be initialized 
+as the [first Step of the SNS launch process](../launching/launch-steps-1proposal.md).
 
 
 This section explains on a technical, but high level, how initial parameters are set and how they affect different canisters.
 The [SNS CLI tool](https://github.com/dfinity/ic/tree/master/rs/sns/cli)
 allows you to set these parameters and includes detailed descriptions of all parameters.
 
 ### Initial parameters
-<!-- NEW: All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production.
-There are some parameters that have to be set and others that are set to a default value but that can also be changed. To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate a given input file. -->
-Apart from the decentralization swap parameters
-(see next section), all SNS parameters are set in a **.yaml** file that can 
-then be passed as an argument
-when installing the SNS, both for testing and in production.
-There are some parameters that have to be set and others that are set to a
-default value but that can also be changed.
-To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate this input file. 
+All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production.
+There are some parameters that have to be set and others that are set to a default value but that can also be changed. To make sure that all parameters are set to valid values, that are also consistent with each other, there is a tool to validate a given input file.
```

Comment:
We don't provide defaults like we did in the last method, all values need to be filled in. We do provide an example one in sns-testing, and we can update it to be like the NNS. Would that be useful to link here?

Also, we do not have the validate command yet, the best means of validating is testing via sns-testing, however we will complete this after we release.

Reply:
* removed the "defaults" part. I don't think we have to link to it here as we do so a few lines further down.

* just to make sure I get it correctly: when I submit a proposal locally (in testing) then the proposal submission would do some validation, right? but there is just not (yet) an additional validation that I can run just over the yaml file without going through the proposal part? I now refrased vaguely to "test the proposal" and propose that we just add more if we add more validation. 

WDYT? Can we resolve this?

Reply:
Confirmed offline. Resolving.